content/en/docs/ops/integrations/spire/index.md

```diff
@@ -212,10 +177,14 @@ Note that `SPIRE Controller Manager` is used in the [quick start](#option-1-quic
         trustDomain: example.org
```

Comment:
odd indent?

Reply:
I _think_ this is correct? I didn't change it. Lemme check how it renders tho.